boards/pyportal/Cargo.toml

```diff
@@ -15,26 +15,43 @@ readme = "README.md"
 edition = "2021"
 
 [dependencies.cortex-m-rt]
-version = "0.7"
+version = "0.7.1"
 optional = true
 
 [dependencies.atsamd-hal]
-version = "0.14"
+version = "0.15.1"
 default-features = false
 
+[dependencies.display-interface-parallel-gpio]
+version="0.6.0"
+optional=true
+
+[dependencies.ili9341]
+version="0.5.0"
+optional=true
+
+[dependencies.usb-device]
+version = "0.2.8"
+optional = true
+
 [dev-dependencies]
 panic-halt = "0.2"
 panic-semihosting = "0.5"
 smart-leds = "~0.3"
```

Comment:
We might want to move this out of dev-dependencies and add some support functions to the bsp. Up to you though.

Reply:
Ugh, github kinda broke my comment, I was referring to `smart-leds`

Reply:
following up on this: we discussed this online and decided not to add this dependency at this point in time